code/game/machinery/doors/shutters.dm

```diff
@@ -6,30 +6,80 @@
 	var/sound_open = 'sound/machines/shutter_open.ogg'
 	var/sound_close = 'sound/machines/shutter_close.ogg'
 	animation_delay = 7
+	var/cut_open = FALSE
+	var/pried_open = FALSE
 
 /obj/machinery/door/poddoor/shutters/preopen
 	icon_state = "shutter0"
 	density = 0
 	opacity = 0
 	layer = BELOW_TABLE_LAYER
 
-/obj/machinery/door/poddoor/shutters/attackby(var/obj/item/weapon/C, var/mob/user)
+/obj/machinery/door/poddoor/shutters/attackby(var/obj/item/I, var/mob/user)
 	add_fingerprint(user)
-	if(!(iscrowbar(C) || (istype(C, /obj/item/weapon/fireaxe) && C.wielded == 1) ))
-		return
-	if(density && (stat & (FORCEDISABLE|NOPOWER)) && !operating)
-		operating = 1
-		spawn(-1)
-			flick("shutterc0", src)
-			icon_state = "shutter0"
-			sleep(animation_delay)
-			plane = open_plane
-			layer = open_layer
-			setDensity(FALSE)
-			set_opacity(0)
-			operating = 0
+	if(istype(I,/obj/item/tool/crowbar/halligan))
+		var/obj/item/tool/crowbar/halligan/H = I
+		if(density) //can't cut an open door open, dumbass
+			if(cut_open)
+				if(!pried_open)
+					pry(user)
+					return
+			else
+				if(!pried_open)
```

Comment:
Both branches check pried open, move this up to the density check

Reply:
resolved in latest commit